server/src/message/data.ts

```diff
@@ -0,0 +1,81 @@
+import { WeatherMessage } from './types';
+
+export const MESSAGES: WeatherMessage[] = [
+  {
+    id: '1',
+    rain_intensity: 'intense',
+    message:
+      'Cuidado {when}, o mundo irá cair! Leve um guarda-chuva, mas talvez precise de um barco',
+  },
+  {
+    id: '2',
+    rain_intensity: 'intense',
+    message: 'Pronto para um dilúvio? {when} tem tempestade chegando',
+  },
+  {
+    id: '3',
+    rain_intensity: 'intense',
+    message:
+      'F*deu, espero que tenha um guarda-chuva contigo, {when} vai chover como nunca antes',
```

Comment:
The messages are awesome, but I think I'm against using curse words 😂 Other thing, we are only supporting English by now, so the messages should have been on it as well. There are places where English and Portuguese are mixed, kinda weird. If you think this changes will take a long time to refactor before we work on the feature itself, we can fix this later on :)